ext_tests/src/lib.rs

```diff
@@ -30,6 +32,51 @@ mod tests {
     fn deserialize_uint() {
         assert_eq!(from_json_str::<UInt>("100").unwrap(), uint!(100));
         assert_eq!(from_json_str::<UInt>("0").unwrap(), uint!(0));
+        assert_eq!(from_json_str::<UInt>("9007199254740991").unwrap(), UInt::MAX);
         assert!(from_json_str::<UInt>("9007199254740992").is_err());
     }
+
+    #[test]
+    #[cfg_attr(feature = "lax_deserialize", ignore)]
+    fn strict_deserialize_int() {
+        assert!(from_json_str::<Int>("-10.0").is_err());
+        assert!(from_json_str::<Int>("-0.0").is_err());
+        assert!(from_json_str::<Int>("0.5").is_err());
+        assert!(from_json_str::<Int>("1.0").is_err());
+        assert!(from_json_str::<Int>("9007199254740991.0").is_err());
+        assert!(from_json_str::<Int>("9007199254740991.49").is_err());
+        assert!(from_json_str::<Int>("9007199254740992.0").is_err());
+    }
+
+    #[test]
+    #[cfg_attr(feature = "lax_deserialize", ignore)]
+    fn strict_deserialize_uint() {
+        assert!(from_json_str::<UInt>("0.5").is_err());
+        assert!(from_json_str::<UInt>("1.0").is_err());
+        assert!(from_json_str::<UInt>("9007199254740991.0").is_err());
+        assert!(from_json_str::<UInt>("9007199254740991.49").is_err());
+        assert!(from_json_str::<UInt>("9007199254740992.0").is_err());
+    }
+
+    #[test]
+    #[cfg_attr(not(feature = "lax_deserialize"), ignore)]
+    fn lax_deserialize_int() {
+        assert_eq!(from_json_str::<Int>("-10.0").unwrap(), int!(-10));
+        assert_eq!(from_json_str::<Int>("-0.0").unwrap(), int!(0));
```

Comment:
I think you just flip-flopped the `#[cfg_attr(not(...` on the wrong test functions.

Reply:
I didn't actually. If this had `#[cfg(not(feature = "lax_deserialize"))]` you would be right. However, this has `#[cfg_attr(not(feature = "lax_deserialize"), ignore)]`, which means the test is going to be ignored (not run) unless the feature is activated.